hail_search/hail_search_query.py

```diff
@@ -168,6 +192,15 @@ def import_filtered_table(self, data_type, sample_data, **kwargs):
 
         return ht
 
+    def _filter_entries_table(self, ht, sample_data):
+        ht = self._add_entry_sample_families(ht, sample_data)
+
+        ht = ht.annotate(family_entries=ht.family_entries.map(
+            lambda entries: hl.or_missing(entries.any(lambda x: x.GT.is_non_ref()), entries))
```

Comment:
I think this filtration is unneeded now that the pipeline is doing this right?

Reply:
no, its still needed because there could be many families in a project but a user has only searched for some of them